### ISSUE TITLE
[Bug]: RHS Pchela-1T drone is un-garageable

### ISSUE THREAD

**Issue by EdWordy** [Antistasi-Ultimate-Community/A3-Antistasi-Ultimate#168]
### Describe the bug

The Pchela-1T drone is ungarage-able due to a bug that says there is still a crew inside (despite being disconnected from the UAV terminal). This is likely on RHS's end but I'm curious what can be done on AU's end. I'm guessing this problem also applies to any other drone added by RHS.

### How to reproduce

1. Load Mods: Antistasi and RHS AFRF
2. Start a fresh Antistasi ultimate game
3. Choose commander role and get a UAV Terminal
4. Enter Zeus
5. Spawn a RHS Pchela-1T Drone
6. Exit Zeus
7. Hack the drone
8. Connect via the UAV terminal
9. Disconnect via the UAV terminal
10. Try to garage the drone

### Version

10.7.0

### Have you altered the code?

No

### What i have changed

```markdown
N/A
```

### Map

Altis

### What server?

Private dedicated server

### Time bug occured (Server time/UTC)

N/A

### Mods

```html
<?xml version="1.0" encoding="utf-8"?>
<html>
  <!--Created by Arma 3 Launcher: https://arma3.com-->
  <head>
    <meta name="arma:Type" content="list" />
    <meta name="generator" content="Arma 3 Launcher - https://arma3.com" />
    <title>Arma 3</title>
    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet" type="text/css" />
    <style>
body {
	margin: 0;
	padding: 0;
	color: #fff;
	background: #000;	
}

body, th, td {
	font: 95%/1.3 Roboto, Segoe UI, Tahoma, Arial, Helvetica, sans-serif;
}

td {
    padding: 3px 30px 3px 0;
}

h1 {
    padding: 20px 20px 0 20px;
    color: white;
    font-weight: 200;
    font-family: segoe ui;
    font-size: 3em;
    margin: 0;
}

em {
    font-variant: italic;
    color:silver;
}

.before-list {
    padding: 5px 20px 10px 20px;
}

.mod-list {
    background: #222222;
    padding: 20px;
}

.dlc-list {
    background: #222222;
    padding: 20px;
}

.footer {
    padding: 20px;
    color:gray;
}

.whups {
    color:gray;
}

a {
    color: #D18F21;
    text-decoration: underline;
}

a:hover {
    color:#F1AF41;
    text-decoration: none;
}

.from-steam {
    color: #449EBD;
}
.from-local {
    color: gray;
}

</style>
  </head>
  <body>
    <h1>Arma 3 Mods</h1>
    <p class="before-list">
      <em>To import this preset, drag this file onto the Launcher window. Or click the MODS tab, then PRESET in the top right, then IMPORT at the bottom, and finally select this file.</em>
    </p>
    <div class="mod-list">
      <table>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">ace</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=463939057" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=463939057</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Antistasi Ultimate - Mod</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=3020755032" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=3020755032</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">BackpackOnChest - Redux</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2372036642" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2372036642</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Better Inventory</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2791403093" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2791403093</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">CBA_A3</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=450814997" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=450814997</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Death and Hit reactions</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2993442344" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2993442344</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Dual Arms - Two Primary Weapons</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1334412770" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1334412770</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Dynamic Camo System</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2800081814" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2800081814</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Enhanced Movement</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=333310405" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=333310405</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">Enhanced Movement Rework</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2034363662" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2034363662</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RE: RHS Escalation</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=2188015838" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=2188015838</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSAFRF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843425103" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843425103</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSGREF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843593391" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843593391</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSPKL</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=1978754337" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=1978754337</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSSAF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843632231" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843632231</a>
          </td>
        </tr>
        <tr data-type="ModContainer">
          <td data-type="DisplayName">RHSUSAF</td>
          <td>
            <span class="from-steam">Steam</span>
          </td>
          <td>
            <a href="https://steamcommunity.com/sharedfiles/filedetails/?id=843577117" data-type="Link">https://steamcommunity.com/sharedfiles/filedetails/?id=843577117</a>
          </td>
        </tr>
      </table>
    </div>
    <div class="footer">
      <span>Created by Arma 3 Launcher by Bohemia Interactive.</span>
    </div>
  </body>
```

**Comment by wersal454** [comment]
Try using new commander button in Y - menu
It forces "crew" outside the vehicles(including UAV crew)

**Comment by EdWordy** [comment]
Forcing the vehicle from the crew seems to have returned the UAV to its original faction, as our AI air defence promptly shot the drone with a ZU-23-2 and blew everyone up in the surrounding area hahaha.

**Comment by wersal454** [comment]
Does this happen only with RHS uavs? Have you tried vanilla UAVs ?

**Comment by wersal454** [comment]
Because I tried with AAF ugv and it worked

**Comment by EdWordy** [comment]
Ill try with the RHS pchela-1T and the vanilla greyhawk, then report back.

**Comment by EdWordy** [comment]
I was unable to reproduce the supposed bug with the pchela. Huh. Weird lol. Im not sure what happenned the previous night when my AI AA defence targeted the drone upon de-crewing.

**Comment by wersal454** [comment]
Yeah, retested the button. Fixed functionality(before, it removed uav driver, then you had to press it again to remove gunner, now you need to press it ones).

**Comment by Silence-112** [comment]
This was fixed in #179
